src/utils/hooks/useForceProjectSelection.ts

```diff
@@ -1,6 +1,7 @@
 import { useEffect } from 'react';
 import { useLocation, useNavigate } from 'react-router-dom-v5-compat';
 
+import { runningTourSignal } from '@kubevirt-utils/components/GuidedTour/utils/constants';
```

Comment:
This is outside the scope of this PR, but I think we should keep signals in their own files and the file name should indicate that it holds signals. They aren't normal constants.